src/range.rs

```diff
@@ -1,7 +1,5 @@
-//! This module contains the parallel iterator types for ranges
-//! (`Range<T>`); this is the type for values created by a `a..b`
-//! expression. You will rarely need to interact with it directly
-//! unless you have need to name one of the iterator types.
+//! Parallel iterator types for [ranges](std::ops::Range), the type
```

Comment:
We'll need to add a markdown link for each of these new links to std we've introduced so they light up in the rendered docs:

```rust
//! Parallel iterator types for [ranges](std::ops::Range), the type
//! for values created by `a..b` expressions.
//! 
//! [std::ops::Range]: https://doc.rust-lang.org/std/ops/struct.Range.html
```